### PR TITLE
feat(zipfile,re): add zipfile & re packages

### PR DESCRIPTION
**Pull request by b5**

### CODE REVIEW

File: zipfile/zip.go [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+	return skylark.NewList(names), nil
+}
+
+func (zf ZipFile) open(thread *skylark.Thread, _ *skylark.Builtin, args skylark.Tuple, kwargs []skylark.Tuple) (skylark.Value, error) {
+	var name skylark.String
+	if err := skylark.UnpackArgs("open", args, kwargs, "name", &name); err != nil {
+		return nil, err
+	}
+	n := string(name)
+	for _, f := range zf.File {
```
Comment (dustmop) [inline zipfile/zip.go]: Oof, unfortunate that this is O(n). Maybe in the future we should build a map from filenames to ZipInfos?

File: re/testdata/test.sky [hunk 2]
```diff
@@ -0,0 +1,18 @@
+load("re.sky", "re")
+load("assert.sky", "assert")
+
+pattern = "(\w*)\s(ADD|REM|DEL|EXT|TRF)\s(.*)\s(NAT|INT)\s(.*)\s(\(\w{2}\))\s(.*)"
+test = "EDM ADD FROM INJURED NAT Jordan BEAULIEU (DB) Western University"
+
+assert.eq(re.match(pattern,test), [("EDM ADD FROM INJURED NAT Jordan BEAULIEU (DB) Western University", "EDM", "ADD", "FROM INJURED", "NAT", "Jordan BEAULIEU", "(DB)", "Western University")])
```
Comment (dustmop) [inline re/testdata/test.sky]: Replace the first part of the right-hand-side with `test`.

File: zipfile/zip.go (hunk 1 above)
Comment (b5) [reply zipfile/zip.go]: agreed. Also agreed on the "in the future" bit 🙂

**Comment by b5** [comment]
Ok cool cool, I know these are super skeletal but the bones are good (pun intended. pun _always_ intended). I'm going to merge this and we'll make it better in the coming weeks.

**Pull request merged**